common/cmake/v2x.cmake

```diff
@@ -118,8 +118,9 @@ function(V2X)
   get_file_target(SRC_TARGET_NAME "${V2X_NAME}.model.xml")
   set_target_properties(${SRC_TARGET_NAME} PROPERTIES INCLUDE_FILES "${MODEL_INCLUDE_FILES}")
 
+  get_rel_target(REL_V2X_NAME v2x ${V2X_NAME})
```

Comment:
This is super wrong.   I really really don't think you know what you are doing.

Reply:
Again, this is the target name which depends on the **all** the files.

Reply:
This changes the meaning of the NAME parameter, but you haven't updated the documentation.  I recommend maybe forging the NAME parameter, and instance rely on TOP_MODULE instead.  Then there is no ambiguity on how NAME is used.  NAME should be for the target NAME, not the input to a function.